artemis-bom/pom.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This new file is seemingly using Windows line endings according to the git diff, can you convert it to Unix line endings to match the rest.